raft/server.cc

```diff
@@ -692,7 +694,7 @@ future<> server_impl::do_on_leader_with_retries(seastar::abort_source* as, Async
 
     while (true) {
         if (as && as->abort_requested()) {
-            throw request_aborted();
+            throw request_aborted("Request aborted while doing on leader with retries");
```

Comment:
Is there something missing in the message? ("while doing" what?)

Perhaps it could be reworded to something like "while retrying on leader" ...